website/emails/newsletter/2026-09-newsletter.html

```
@@ -0,0 +1,2119 @@
+												<p
+													class="feature"
+													style="
+														margin: 20px 0 10px 0;
+														font-family: 'Unica77', Helvetica, Arial, sans-serif;
+														font-weight: 400;
+														font-size: 22px;
+														line-height: 30px;
+														color: #191a19;
+													"
+												>
+													<span style="color: #01579b">No more scented candles and books that go straight to the shelf.
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </span> ], start tag match failed [ <span style="color: `#01579b`"> ] on line 1177.

(tag-pair)